src/main/java/sx/blah/discord/handle/obj/IGuild.java

```diff
@@ -541,6 +541,19 @@
 	 * @return The emoji with the given name (or null if one was not found).
 	 */
 	IEmoji getEmojiByName(String name);
+	
+	/**
+ 	 * This adds a custom emoji to the guild.
+ 	 *
+ 	 * @param name The name, <b>without colons</b> of length 2-32 characters.
+ 	 * @param image The Image of the Emoji.
+ 	 * @return The emoji.
+ 	 *
+ 	 * @throws DiscordException
+ 	 * @throws RateLimitException
+ 	 * @throws MissingPermissionsException
+ 	 */
+ 	IEmoji addEmoji(String name, Image image);
```

Comment:
Should be `createEmoji` like `createChannel`. Should also have a `roles` method.